fabric-content-registries-v1/src/main/java/net/fabricmc/fabric/impl/content/registries/FuelRegistryImpl.java

```diff
@@ -24,15 +24,17 @@
 
 import net.fabricmc.fabric.api.content.registry.v1.FuelRegistry;
 
+@Deprecated
```

Comment:
i request you put a message with the @Deprecated to tell the dev what the replacement is

Reply:
this in is impl, not api